Setup.sh

```diff
@@ -1,144 +1,94 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
-# EXPERIMENTAL!
-# This file is experimental and is not maintained directly by Improbable. Please use at your own risk.
-
-set -e -u -o pipefail
+set -e -u -o pipefail -x
```

Comment:
Not a bash expert, but I think the -x will cause the script to print all the commands it is executing. Is this what we want?

Reply:
changed it based on Jamie's suggestion to only do it optionally as it is helpful for debugging sometimes